tests/integration-tests/tests/basic/test_essential_features.py

```diff
@@ -66,3 +231,32 @@ def _test_custom_bootstrap_scripts_args_quotes(cluster):
     """
     # Check head node and compute node status
     check_status(cluster, "CREATE_COMPLETE", head_node_status="running", compute_fleet_status="RUNNING")
+
+
+def _test_hit_disable_hyperthreading(
```

Comment:
minor: you can remove the "hit" word, this was an old term to distinguish between: "Heterogeneous instance types" and "Single instance type". but we don't need it anymore.